Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -80,12 +85,14 @@
   private String mSelectedGameId;
   private int mPlatform;
   private String[] mPaths;
+  private static boolean sUserPausedEmulation;
```

Comment:
This should not be static.

Reply:
`EmulationActivity.hasUserPausedEmulation()` uses `sUserPausedEmulation` and is called in EmulationFragment so don't we need a static variable for `EmulationActivity.hasUserPausedEmulation()`?

Reply:
You should be able to use `activity.hasUserPausedEmulation()` in EmulationFragment.

Reply:
But in EmulationFragment, class `EmulationState` is static and `hasUserPausedEmulation()` is called within that class. `activity` isn't static (and can't be static) so wouldn't my code be the best approach here?

Reply:
Ah, I had missed that. But can't we just make EmulationState non-static, or pass in `activity` when creating an EmulationState?

It is preferable to not use static state in a way that would cause things to break when creating two instances of a class at the same time, even if we never actually do that with EmulationActivity.

Reply:
Let's be safe and pass `activity`